plugins/ingress_nginx_logs.yaml

```diff
@@ -0,0 +1,102 @@
+version: 0.0.1
+title: Ingress NGINX
+description: Log parser for Ingress NGINX
+parameters:
+  - name: file_path
+    type: "[]string"
+    default:
+      - "/var/log/containers/ingress-nginx-controller*.log"
+  - name: start_at
+    type: string
+    supported:
+      - beginning
+      - end
+    default: end
+template: |
+  receivers:
+      filelog:
+        include:
+          {{ range $fp := .file_path }}
+          - '{{ $fp }}'
+          {{ end }}
+        start_at: {{ .start_at }}
+        poll_interval: 500ms
```

Comment:
Should this be hard coded or configurable?

Reply:
No, removed.